replicated.tf

```diff
@@ -1,15 +1,15 @@
 locals {
   rptfeconf = {
-    "demo"      = "${file("${path.module}/data/demo.json")}"
-    "airgap"    = "${file("${path.module}/data/airgap.json")}"
-    "es"        = "${file("${path.module}/data/es.json")}"
-    "es_airgap" = "${file("${path.module}/data/es_airgap.json")}"
+    "demo-online"              = "${file("${path.module}/data/demo.json")}"
```

Comment:
at some point(not GA) I think we could leverage terraform's templating language better with these

Reply:
Probably, yeah.